lib/AST/Decl.cpp

```diff
@@ -2082,7 +2082,8 @@ SourceLoc ValueDecl::getAttributeInsertionLoc(bool forModifier) const {
 /// Returns true if \p VD needs to be treated as publicly-accessible
 /// at the SIL, LLVM, and machine levels due to being @usableFromInline.
 bool ValueDecl::isUsableFromInline() const {
-  assert(getFormalAccess() == AccessLevel::Internal);
+  if (getFormalAccess() != AccessLevel::Internal)
+    return false;
```

Comment:
Unhappy with this. If you're going to make this an API that can be called everywhere, shouldn't it return true for `open` and `public` decls as well?